elfi/methods/methods.py

```diff
@@ -922,41 +952,147 @@ class BOLFI(InferenceMethod):
     http://jmlr.org/papers/v17/15-017.html
 
     """
-
-    def __init__(self, model, batch_size=1, discrepancy=None, bounds=None, **kwargs):
+    def __init__(self, model, target=None, outputs=None, batch_size=1,
+                 initial_evidence=10, update_interval=10, bounds=None, target_model=None,
+                 acquisition_method=None, acq_noise_cov=1., **kwargs):
         """
         Parameters
         ----------
         model : ElfiModel or NodeReference
-        discrepancy : str or NodeReference
+        target : str or NodeReference
             Only needed if model is an ElfiModel
-        discrepancy_model : GPRegression, optional
+        target_model : GPyRegression, optional
```

Comment:
Maybe we could say here that in BOLFI the target model is the discrepancy model.